### ISSUE TITLE
Add ftm network

### ISSUE THREAD

**Issue by DarkGhost7** [Trackavault/trackavault-ui#2]
add fantom network for prod vaults on fantom

**Comment by Deams51** [comment]
Been working on it, but would need to find a full node running in archive mode to be able to release it. 
Do you know any @DarkGhost7 ? Been using alchemy for the eth network, but doesn't seem like they'll be releasing the same service for other networks any time soon.

**Comment by DarkGhost7** [comment]
Yes I host one you can use can you reach out to me on telegram and we can talk / in the yearn chats.
